cluster/local/util.sh

```diff
@@ -21,6 +21,6 @@ function prepare-e2e() {
 function detect-master {
   KUBE_MASTER=localhost
   KUBE_MASTER_IP=127.0.0.1
-  KUBE_MASTER_URL="http://${KUBE_MASTER_IP}:8080"
+  export KUBE_MASTER_URL="http://${KUBE_MASTER_IP}:8080"
```

Comment:
hmm why do we need to `export`?

Reply:
IIRC this is sourced and then this env is read by other scripts because :cluster/:. not 100% though

Reply:
if `KUBE_MASTER_URL` is needed by other scripts, `shellcheck` suggest we export it. Otherwise we can remove this env if it's safe.

Reply:
Yeah, `hack/ginkgo-e2e.sh` should read this env from cluster/loca/util.sh for local provider case like
`KUBE_MASTER_URL="${KUBE_MASTER_URL:-https://${KUBE_MASTER_IP:-}}"` but it cannot do that at this time and `hack/ginkgo-e2e.sh` uses a hard-coded default value.
It seems nice to add `export` for this case.

Reply:
Can you add a comment to the above function explaining what it requires, sets and exports? eg:
```
# Detect the IP for the master
#
# Vars set:
#   KUBE_MASTER
#   KUBE_MASTER_IP
# Vars exported:
#   KUBE_MASTER_URL
function detect-master {
```

It may seem redundant but it helps us know this is on purpose, and it's the same style used in `cluster/gce`

Reply:
@spiffxp sure, PR updated, PTAL